packages/manager/apps/pci-volume-snapshot/src/pages/listing/Status.component.tsx

```diff
@@ -0,0 +1,48 @@
+import { OdsBadgeColor } from '@ovhcloud/ods-components';
+import { OdsBadge } from '@ovhcloud/ods-components/react';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { TSnapshot } from '@/api/data/snapshots';
+
+type StatusComponentProps = {
+  status: TSnapshot['status'];
+};
+
+export default function StatusComponent({
+  status,
+}: Readonly<StatusComponentProps>) {
+  const { t } = useTranslation('volumes');
+
+  const statusGroup = ['error', 'error_deleting'].includes(status)
+    ? 'ERROR'
+    : status.toUpperCase();
+
+  const [badgeColor, setBadgeColor] = useState<OdsBadgeColor>('information');
```

Comment:
i think you can replace the useState + useEffect with an useMemo, since having a state for badgeColor is not necessary here